### PR TITLE
Addfixed new iso6393 codes to extant lgs, see issue #264

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/indo1319/germ1287/nort3152/nort3160/nort3266/narr1283/jamt1238/md.ini [hunk 1]
```diff
@@ -2,6 +2,7 @@
 [core]
 name = Jamtska
 hid = jmk
+iso639-3 = jmk
```
Comment (xrotwang) [inline languoids/tree/indo1319/germ1287/nort3152/nort3160/nort3266/narr1283/jamt1238/md.ini]: `jmk` is not a valid ISO 639-3 code!

File: languoids/tree/indo1319/germ1287/nort3152/nort3160/sout3248/scan1238/md.ini [hunk 2]
```diff
@@ -3,6 +3,7 @@
 name = Scanian
 hid = scy
 level = language
+iso639-3 = scy
```
Comment (xrotwang) [inline languoids/tree/indo1319/germ1287/nort3152/nort3160/sout3248/scan1238/md.ini]: invalid ISO code

File: languoids/tree/sino1245/burm1265/naqi1236/qian1263/rgya1241/core1262/situ1238/md.ini [hunk 3]
```diff
@@ -3,6 +3,7 @@
 name = Situ
 hid = tzi
 level = language
+iso639-3 = tzi
```
Comment (xrotwang) [inline languoids/tree/sino1245/burm1265/naqi1236/qian1263/rgya1241/core1262/situ1238/md.ini]: invalid ISO code

File: languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/yang1304/md.ini [hunk 4]
```diff
@@ -3,6 +3,7 @@
 name = Yangliu
 hid = lly
 level = language
+iso639-3 = lly
```
Comment (xrotwang) [inline languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/yang1304/md.ini]: invalid ISO code

File: languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/grea1292/xuzh1234/md.ini [hunk 5]
```diff
@@ -3,6 +3,7 @@
 name = Xuzhang Lalo
 hid = lxu
 level = language
+iso639-3 = lxu
```
Comment (xrotwang) [inline languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/grea1292/xuzh1234/md.ini]: invalid ISO code

File: languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/ekaa1234/md.ini [hunk 6]
```diff
@@ -3,6 +3,7 @@
 name = Eka
 hid = ekb
 level = language
+iso639-3 = ekb
```
Comment (xrotwang) [inline languoids/tree/sino1245/burm1265/lolo1265/lolo1267/nili1235/liso1234/nucl1734/lisu1252/lalu1234/lalo1240/ekaa1234/md.ini]: invalid ISO code

**Comment by d97hah** [comment]
I am an old dog, unable to remember to run glottolog check, but I will try

**Comment by xrotwang** [comment]
ok, looks good now. But while we're at it, do we want to address any of these:
```
WARNING Palumata [palu1237]: <ISO-639-3 [pmc] Retirement/duplicate> changed to [huw]
WARNING Talur [talu1239]: <ISO-639-3 [ilw] Retirement/merge> changed to [gal]
WARNING Kota Marudu Tinagas [kota1278]: <ISO-639-3 [ktr] Retirement/merge> changed to [dtp]
WARNING Coastal Kadazan [coas1294]: <ISO-639-3 [kzj] Retirement/merge> changed to [dtp]
WARNING Eastern Gurung [east2345]: <ISO-639-3 [ggn] Retirement/merge> changed to [gvr]
WARNING Tempasuk Dusun [temp1235]: <ISO-639-3 [tdu] Retirement/merge> changed to [dtp]
WARNING Moldavian [mold1248]: <ISO-639-3 [mol] Retirement/merge> changed to [ron]
WARNING Jeng [jeng1241]: <ISO-639-3 [jeg] Retirement/merge> changed to [oyb]
WARNING Navarro-Labourdin Basque [basq1249]: <ISO-639-3 [bqe] Retirement/merge> changed to [eus]
WARNING Tambunan Dusun [tamb1254]: <ISO-639-3 [kzt] Retirement/merge> changed to [dtp]
WARNING Sok [sokk1239]: <ISO-639-3 [skk] Retirement/merge> changed to [oyb]
WARNING Mirpur Panjabi [mirp1238]: <ISO-639-3 [pmu] Retirement/merge> changed to [phr]
WARNING Chimakuan [chim1311]: <ISO-639-3 [cmk] Retirement/duplicate> changed to [xch]
WARNING Karahawyana [kara1475]: <ISO-639-3 [xkh] Retirement/duplicate> changed to [waw]
WARNING Ndaktup [ndak1242]: <ISO-639-3 [ncp] Retirement/duplicate> changed to [kdz]
WARNING Tai Mène [taim1250]: <ISO-639-3 [tmp] Retirement/merge> changed to [tyj]
WARNING Mea [meaa1237]: <ISO-639-3 [meg] Retirement/merge> changed to [cir]
WARNING Xiandao [xian1249]: <ISO-639-3 [xia] Retirement/merge> changed to [acn]
WARNING Krim [krim1238]: <ISO-639-3 [krm] Retirement/merge> changed to [bmf]
WARNING Darkhat [dark1243]: <ISO-639-3 [drh] Retirement/merge> changed to [khk]
WARNING Tinoc Kallahan [tino1235]: <ISO-639-3 [tne] Retirement/merge> changed to [kak]
```

**Comment by d97hah** [comment]
Maybe. There are some classes of cases in there:

- Retired iso:s that are in bookkeeping or unattested in Glottolog: IMHO
this is desired behaviour and shouldn't generate a warning in glottolog
check
- Retired iso:s that are mergers but show up as dialects in Glottolog: IMHO
this is desired behaviour and shouldn't generate a warning in glottolog
check
- Iso changes where I "don't agree that ISO should have made that change on
their own principles": Not sure what should be desired behaviour
- Iso changes where I "haven't had the time to check up on it, could be due
to inertia or due to waiting time to get a hold of a relevant document":
These should generate the warning, as is.

The first two categories can be distinguished by a computer but not the
latter two.

all the best, H


Pada tanggal Jum, 26 Okt 2018 pukul 08.59 Robert Forkel <
notifications@github.com> menulis:

**Comment by xrotwang** [comment]
Ah, good point about the case of languoids in bookkeeping - will adapt the check!

**Pull request merged**